docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+
+volumes:
+  database2: {}
+
+services:
+  # Named "database2" because "database" conflicts with directory inside project
```

Comment:
Why should they conflict?

Reply:
No clue, it didn't work with `database`, but it worked fine with `database2`..